### ISSUE TITLE
automatic story list

### ISSUE THREAD

**Issue by Zambonifofex** [Zambonifofex/stories#8]
After resolving #2, a pretty important “followup issue” is that the story list (the landing page) isn’t generated automatically. So, although the stories are on GCS, and can become accessible immediately on the website (i.e. without a redeployment), the story list is still static and needs to be updated manually and redeployed.

An interesting thing to note is that this is blocked by #3, as it’s important to allow “unpublished” stories that don’t appear on the story list, but can be accessed through the website.
